src/Form/Form.php

```diff
@@ -209,12 +209,22 @@ protected function _buildSchema(Schema $schema): Schema
      * Used to check if $data passes this form's validation.
      *
      * @param array $data The data to check.
+     * @param string|bool $validator Validator name or `true` for default validator.
      * @return bool Whether or not the data is valid.
+     * @throws \RuntimeException If validator is invalid.
      */
-    public function validate(array $data): bool
+    public function validate(array $data, $validator = true): bool
     {
-        $validator = $this->getValidator();
-        $this->_errors = $validator->validate($data);
+        if (!$validator) {
+            return true;
```

Comment:
I feel like this should return false so that calls with `$validator == false` which also includes `null` (clearly a developer mistake) fail validation instead of passing. Having the method succeed when there is obvious user error seems bad to me.

Another option could be to throw an exception when `$validator === null`. This would enforce the doc block type at runtime helping catch mistakes earlier.

Reply:
$validator = false (or null) mean "no validation", the method should pass.